src/jvm/test/resources/expected_logs/obstruction_freedom_violation_events_cut.txt

```diff
@@ -10,12 +10,12 @@ The following interleaving leads to the error:
 |                                                      Thread 1                                                       |                                                      Thread 2                                                      |
 | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 |                                                                                                                     | enqueue(-1)                                                                                                        |
-|                                                                                                                     |   q.enqueue(-1) at ObstructionFreedomViolationEventsCutTest.enqueue(SpinlockEventsCutTests.kt:34)                  |
+|                                                                                                                     |   q.enqueue(-1) at ObstructionFreedomViolationEventsCutTest.enqueue(SpinlockEventsCutTests.kt:33)                  |
```

Comment:
Is there any difference besides the line numbers? Please check other outputs as well.